src/app/pages/administration/jobs/scheduled/trigger.html

```diff
@@ -12,7 +12,20 @@
       <label for="{{ param.key }}" >{{ param.label }}</label>
     </div>
     <div class="col-md-6">
-      <input class="input-sm form-control" type="text" id="{{ param.key }}" name="{{ param.key }}" ng-model="$ctrl.trigger.job.jobDataMap[param.key]" required>
+      <input class="input-sm form-control" type="text" id="{{ param.key }}" name="{{ param.key }}" ng-model="$ctrl.trigger.job.jobDataMap[param.key]" required ng-if="!param.type || param.type == 'text'">
```

Comment:
```suggestion
      <input class="input-sm form-control" type="text" id="{{ param.key }}" name="{{ param.key }}" ng-model="$ctrl.trigger.job.jobDataMap[param.key]" required ng-if="!param.type || param.type === 'text'">
```

I know there's a difference in javascript for `==` vs `===` and `!=` vs `!==`, and think it has something to with types of equality, though I can't remember specifically what it was. That said, the best practices suggest using the three character versions almost exclusively

Similar comments to lines below

Reply:
I think "===" also checks that the data type is the same
1 == '1'  -> true
1 === '1' -> false